.github/workflows/ci.yml

```diff
@@ -13,38 +13,29 @@ jobs:
   test-bash:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
       - name: Run tests
         run:  make test_bashscripts
 
-  build-start-test:
+  unit-tests:
```

Comment:
Needs to use `make test_unit` instead of `make test`

Reply:
Called non-search following discussion on slack